assets/css/main.css

```diff
@@ -71,9 +71,6 @@ input, select, textarea {
 	-ms-appearance: none;
 	appearance: none;
 }
-#main {
-	margin-top: 10em;
-}
```

Comment:
Instead of removing #main style, change
`margin-top: 10em;` to `padding: 4em`

Reply:
![Screenshot 2021-04-19 142024](https://user-images.githubusercontent.com/60707646/115208708-7d662580-a11a-11eb-80d0-349a60e32ca1.png)